src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -454,8 +454,14 @@ export class OperationDetails {
     }
 
     private scrollToOperation() {
-        const headerElement = document.getElementById("operation-name");
-        headerElement && headerElement.scrollIntoView({ behavior: "smooth", block: "start", inline: "start" });
+        if (this.routeHelper.getDefinitionName()) {
+            const definitionId = this.router.getHash();
```

Comment:
Here, I think the code would be cleaner if we use the if statement just for deciding where the scroll should happen, and then do the actual scrolling. Something like:


```
let scrollTo : HTMLElement;

if (this.routeHelper.getDefinitionName()) {
    const definitionId = this.router.getHash();
    scrollTo = document.getElementById(definitionId);
} else {
    scrollTo = document.getElementById("operation-name");
}

scrollTo && scrollTo.scrollIntoView({ behavior: "smooth", block: "start", inline: "start" });
    
```